### ISSUE TITLE
On Wadsworth Street between Euclid Avenue and Fort Avenue, motor vehicles speed excessively, creating a loud and dangerous environment

### ISSUE THREAD

**Issue by partytax** [partytax/lynchburg-road-issues#159]
https://www.openstreetmap.org/#map=19/37.40161/-79.16365

The official way to resolve residential through-street speeding issues is through a Neighborhood Traffic Management Petition (NTMP): https://www.lynchburgva.gov/DocumentCenter/View/1292/Neighborhood-Traffic-Management-Program-NTMP-PDF

Saved PDF: [Neighborhood Traffic Management Program (NTMP) (PDF).pdf](https://github.com/Lynchburg-Urban-Network/lynchburg-road-issues/files/14172425/Neighborhood.Traffic.Management.Program.NTMP.PDF.pdf)

**Comment by partytax** [comment]
Emailed the urban forester to request a meeting to discuss filling in some street trees. In time, the sense of enclosure created by these trees should discourage extreme speeding.

```
Good afternoon. Would you be willing to walk Wadsworth Street with me sometime in the next month to discuss filling in some missing street trees? In time, these trees should create a sense of enclosure around the street and discourage some of the extreme speeding I've witnessed.

I'm guessing you work out of the Miller Center offices, so this could be a pretty convenient field trip!

Thank you for your consideration.

Nathan
```

Not marking as reported yet, because the agency with the most power to affect this issue (public works) has not been notified.

**Comment by partytax** [comment]
Followed up with email to Nicholas Meyer:

> Mr. Meyer,
> 
> Just following up on this message I sent earlier this year. If you have 15 minutes to meet RE shade tree fill-in on Wadsworth Street, I'd greatly appreciate it.
> 
> Nathan

**Comment by partytax** [comment]
Made personal task to decide scope of Neighborhood Traffic Management petition and doodle out some designs for 15 minutes.

**Comment by partytax** [comment]
Received response from urban forester:

>  Hey Nathan,
> 
> Thanks for following up!
> I'd be glad to meet.
> My availability this week is generally 9:00am-4:15pm every day. Let me know when may be good for you.
> 
> Thanks!
> 
> Nicholas Meyer
> ISA Board Certified Master Arborist MA-5720B
> ISA TRAQ
> Urban Forester
> City of Lynchburg
> Mobile 434-209-4716
> Nicholas.Meyer@LynchburgVa.gov

**Comment by partytax** [comment]
Annotated map with blue plus symbols to indicate places where it'd be nice to have a tree

![wadsworth-trees](https://github.com/Lynchburg-Urban-Network/lynchburg-road-issues/assets/29897671/40072f35-39bc-41f7-93b8-343bb062ef69)

**Comment by partytax** [comment]
Responded to urban forester:

> How about 12:30p tomorrow (the 14th)? Would you prefer that I meet at your office or at Wadsworth Street? If your office, would that be on Memorial Avenue at Public Works building?
> 
> I've attached an [ambitious] map with blue symbols where it would be awesome to have a shade tree. The green circles are existing trees, the dashed red lines are sidewalks, and the gray dots and lines are power poles and lines.
> 
> Nathan

**Comment by partytax** [comment]
About 53 parcels along Wadsworth Street between Euclid Avenue and Marsh Street. I know about 10 of the property owners. Perhaps should extend scope to Fort Avenue because that's where the speeding starts.

**Comment by partytax** [comment]
Collected signatures and submitted NTMP form to traffic engineer through certified mail. Waiting on response.

**Comment by partytax** [comment]
On Monday morning (2024-07-29) Public Works Engineering installed pneumatic tube vehicle counter (and speed measurer) at 37.402161, -79.164925 in front of Miller Center. On Friday morning (2034-08-02) they gathered data and removed the device. 4 days of data. 

They said they could not run it over the weekend because they must check device every day and people are off on the weekend.

**Comment by partytax** [comment]
Received a letter about a week ago informing that there is not a documented speeding problem according to the city engineering criteria. In step 2b of the NTMP process, you have to get 40 points in table 1 before being evaluated by table 2.

I scored based on the 1916 vehicle per day and 29 mph 85th percentile speed they gathered from their instruments.

We got 38.32 points out of 40 required for table 2, so did not move forward.

Out of curiosity, I did the scoring for table 2 and got 45-55 points (40 required), depending on whether you count Miller Park and Park View Community Mission as community institutions generating significant pedestrian traffic.

[ntmp-section-2b-calculations.ods](https://github.com/user-attachments/files/16698867/ntmp-section-2b-calculations.ods)

**Comment by partytax** [comment]
Letter ![signal-2024-08-22-210910_002](https://github.com/user-attachments/assets/09af703c-3b87-4aef-8eb0-4f1f01ed8d55)
Our returned request page \#1 ![signal-2024-08-22-210943_002](https://github.com/user-attachments/assets/932c8c5e-b1ac-4651-b57c-fe09177f45c3)
Our returned request page \#2 ![signal-2024-08-22-211010_002](https://github.com/user-attachments/assets/6457c8f9-cf9e-4e9f-8710-808c5e59111b)
Speed readout ![signal-2024-08-22-211049_002](https://github.com/user-attachments/assets/94ec9933-bf06-4e95-a174-be93e47fb494)

**Comment by partytax** [comment]
switching to proposal

**Comment by partytax** [comment]
Traffic engineer sent me links to UrbanSDK dashboard that shows vehicle speeds over the month of April based on data from peoples' phones. Screenshotting because dashboard link will expire. Max speeds recorded are 70 and 69 mph for two sections of Wadsworth Street. 95th percentile speeds aren't too bad at 28 and 26 mph. I think what makes the street suck is the fastest 1-2% of drivers.

<img width="652" height="588" alt="Image" src="https://github.com/user-attachments/assets/beb94f37-ad62-43ad-8e60-f88374aa07fc" />
<img width="675" height="627" alt="Image" src="https://github.com/user-attachments/assets/a88b4e8c-d630-4b27-a237-5cabcac2b473" />
